project-koku/koku
# 1933 wrong stats months

#1933 

This PR flips the order_by `billing_period_start_datetime` from ascending to descending which will return the 2 most recent months to the /sources/{id}/stats/ endpoint.

Testing:
1. Add OCP source
2. Upload more than 2 months of data using Nise
3. Watch stats for provider
4. See 2 most recent months.
